packages/rrweb/src/record/mutation.ts

```diff
@@ -520,7 +523,7 @@ export default class MutationBuffer {
                 this.maskTextSelector,
               ) && value
                 ? this.maskTextFn
-                  ? this.maskTextFn(value)
+                  ? this.maskTextFn(value, el)
```

Comment:
Lets do this inline so it doesn't get called if this.maskTextFn is false.
These get called often and can create performance issues, so I'd prefer we don't call it if it isn't absolutely necessary
```suggestion
                  ? this.maskTextFn(value, getElementFromNode(m.target))
```

Reply:
Ah that's a great catch.